applications/GeoMechanicsApplication/custom_processes/apply_k0_procedure_process.cpp

```diff
@@ -43,37 +43,65 @@ void SetConsiderDiagonalEntriesOnlyAndNoShear(ModelPart::ElementsContainerType&
 namespace Kratos
 {
 
-ApplyK0ProcedureProcess::ApplyK0ProcedureProcess(ModelPart& model_part, Parameters K0Settings)
-    : Process(Flags()), mrModelPart(model_part), mSettings(std::move(K0Settings))
+ApplyK0ProcedureProcess::ApplyK0ProcedureProcess(Model& rModel, const Parameters& K0Settings)
```

Comment:
To be consistent with the Kratos Style guide, we should append an `r` to this function parameter name:
```suggestion
ApplyK0ProcedureProcess::ApplyK0ProcedureProcess(Model& rModel, const Parameters& rK0Settings)
```

Reply:
Reverted to passing by value